### PR TITLE
Broadening Wording of Abstract and Introduction

### PR DESCRIPTION
**Pull request by henkbirkholz**
A proposal for discussion.

### CODE REVIEW

**Comment by henkbirkholz** [comment]
I'd recommend to restructure the markdown to one line per sentence before the next submission so that github changes are visualized more comprehensive.

File: draft-dthaler-rats-endorsements.md [hunk 1]
```diff
@@ -39,37 +39,43 @@ informative:
...
 Attestation Results) involve comparing the actual state of an Attester against
-desired or undesired states, in order to determine how trustworthy the Attester
-is for its purposes.  Thus, a Verifier needs to receive messages with information
-about actual state, and information about desired/undesired states, and an appraisal
-policy that controls how the two are compared.
-
-Each Attester in general has multiple Target Environments (e.g., hardware, firmware,
-Operating System, etc.), each with their own set of claims (sometimes called
-a "claimset").
+desired or undesired states , in order to determine how trustworthy the Attester
```
Comment (dthaler) [inline draft-dthaler-rats-endorsements.md]:
```suggestion
desired or undesired states, in order to determine how trustworthy the Attester
```

File: draft-dthaler-rats-endorsements.md [hunk 2]
```diff
@@ -39,37 +39,43 @@ informative:
...
+is for its purposes.  The state of an Attester represents it's composition of
```
Comment (dthaler) [inline draft-dthaler-rats-endorsements.md]:
```suggestion
is for its purposes.  The state of an Attester represents its composition of
```

File: draft-dthaler-rats-endorsements.md [hunk 3]
```diff
@@ -39,37 +39,43 @@ informative:
...
+components of execution environments (its "shape"), typically in a hierarchical
+manner.  The state of an Attester also encompasses the combination of static and
+dynamic constitution (e.g., provisioned and deployed software, firmware, and
+micro-code), static and dynamic configuration, and the resulting operational state
+of its components at a certain point of time. Thus, a Verifier needs to receive
+messages with information about actual state, and information about desired/undesired
+states, and an appraisal policy that controls how the two are compared.
+
+Each Attester in general has at least one Attesting Environment and one Target
+Environment (e.g., hardware, firmware, Operating System, etc.).  Typically, a each
```
Comment (dthaler) [inline draft-dthaler-rats-endorsements.md]:
```suggestion
Environment (e.g., hardware, firmware, Operating System, etc.).  Typically, each
```

File: draft-dthaler-rats-endorsements.md [hunk 4]
```diff
@@ -39,37 +39,43 @@ informative:
...
+Attester has multiple Target Environments, each with their own set of claims (sometimes
+called a "claimset") representing their actual state.  Additionally, the number of
+target Environments is not limited.
```
Comment (dthaler) [inline draft-dthaler-rats-endorsements.md]:
```suggestion
Target Environments is not limited.
```

**Review by dthaler — APPROVED** [review]
Good other than some typos.
Thanks!

**Pull request merged**